### PR TITLE
Miguelhar.plat 6528 OIDC

### PR DESCRIPTION
**Pull request by miguelhar**
[PLAT-6528](https://dominodatalab.atlassian.net/browse/PLAT-6528)

### CODE REVIEW

File: submodules/eks/cluster.tf [hunk 1]
```diff
@@ -72,6 +72,17 @@ resource "aws_eks_cluster" "this" {
   }
 }
 
+data "tls_certificate" "cluster_tls_certificate" {
+  url = aws_eks_cluster.this.identity[0].oidc[0].issuer
```
Comment (ddl-dclegg) [inline submodules/eks/cluster.tf:76]: Can we do something like `try(aws_eks_cluster.this.identity[0].oidc[0].issuer, "")` here? In some recent operations I've seen index errors on things like this when the reference/attribute is null.

**Comment by fraenkel** [comment]
This just seems to be the first part of the actual task. You have enabled OIDC to EKS.
How does one configure the mapping?

**Comment by miguelhar** [comment]
Thats we discussed we were doing. Thats up to the consumer to set up for the time being. While we figure out the structure.

File: submodules/eks/cluster.tf (hunk 1 above)
Comment (miguelhar) [reply submodules/eks/cluster.tf:76]: Thats a property on eks I dont expect it to ever change and if it does I rather it blows up.

**Review by steved — APPROVED** [review]
One question, but LGTM as a first step.

File: submodules/eks/cluster.tf (hunk 1 above)
Comment (ddl-dclegg) [reply submodules/eks/cluster.tf:76]: I'm not sure I agree, but I'm not going to hold things up because of that.

**Review by ddl-dclegg — APPROVED** [review]

**Pull request merged**